LowerBodyModule.py

```
@@ -2,6 +2,7 @@
 import time
 import numpy as np
 import pose_estimation_module as pem
+import nothing
```

Ruff (0.8.0)

`nothing` imported but unused

Remove unused import: `nothing`

(F401)